src/ElmFormat/Execute.hs

```diff
@@ -22,9 +23,19 @@ forHuman operation =
 
 
 {-| Execute Operations in a fashion appropriate for use by automated scripts. -}
-forMachine :: ElmVersion -> OperationF a -> IO a
+forMachine :: ElmVersion -> OperationF a -> StateT Bool IO a
 forMachine elmVersion operation =
     case operation of
-        InFileStore op -> FileStore.execute op
+        InFileStore op -> lift $ FileStore.execute op
         InInfoFormatter op -> Json.format elmVersion op
-        DeprecatedIO io -> io
+        DeprecatedIO io -> lift io
+
+
+forMachineInit :: (IO (), Bool)
```

Comment:
Yeah, I was thinking about that too, but didn't have a clear answer, so decided to leave it TBD for now.